docs/options.md

```diff
@@ -395,6 +397,21 @@ Type: `string | "current" | true`.
 
 If you want to compile against the current node version, you can specify `"node": true` or `"node": "current"`, which would be the same as `"node": process.versions.node`.
 
+Alternatively, you can specify node version in a browserslist query:
+
+```jsonc
+{
+  "targets": "node 12" // not recommended
+}
+```
+
+In this case, browserslist will resolve it to the _latest_ version available in the `node-releases` library. Because Node.js may support new language features in minor releases, a program generated for Node.js 12.22 may throw a syntax error on Node.js 12.0. We recommend you always specify a minor version when using browserslist node queries:
```

Comment:
```suggestion
In this case, browserslist will resolve it to the _latest_ version available in the `node-releases` library. Because Node.js may support new language features in minor releases, a program generated for Node.js 12.22 may throw a syntax error on Node.js 12.0. We recommend that you always specify a minor version when using node queries with browserslist:
```